mlrun/db/httpdb.py

```diff
@@ -3365,6 +3366,40 @@ def delete_project_secrets(
             params=params,
         )
 
+    def get_model_endpoint_monitoring_metrics(
+        self,
+        project: str,
+        endpoint_id: str,
+        type: Literal["results", "metrics", "all"] = "all",
+    ) -> list[mm_endpoints.ModelEndpointMonitoringMetric]:
+        """
+        :param project: The name of the project.
+        :param endpoint_id: The unique id of the model endpoint.
+        :param type: The type of the metrics to return. "all" means "results" and "metrics".
+        :return: A list of the application metrics or/and results for this model endpoint.
+
+        """
+        path = f"projects/{project}/model-endpoints/{endpoint_id}/metrics"
+        params = {"type": type}
+        error_message = (
+            f"Failed to get model endpoint monitoring metrics,"
+            f" endpoint_id: {endpoint_id}, project: {project}"
+        )
+        response = self.api_call(
+            mlrun.common.types.HTTPMethod.GET,
+            path,
+            error_message,
+            params=params,
+        )
+        monitoring_metrics = response.json()
+        if monitoring_metrics:
+            return [
+                mm_endpoints.ModelEndpointMonitoringMetric(**monitoring_metric)
+                for monitoring_metric in monitoring_metrics
+            ]
+        else:
+            return []
```

Comment:
1. Consider using `pydantic.parse_obj_as`:
   https://stackoverflow.com/a/61021183/27658872
   Just a recommendation, up to you.
2. The if-else is probably unnecessary as `monitoring_metrics = []` returns the expected result.